src/backend/src/utils/users.utils.ts

```diff
@@ -1,7 +1,11 @@
-import { User } from '@prisma/client';
+import { User, User_Settings } from '@prisma/client';
 import prisma from '../prisma/prisma';
 import { HttpException, NotFoundException } from './errors.utils';
 
+type UserWithSettings = ({
```

Comment:
UsersWithSettings

Reply:
actually make this not be an array, keep it as just UserWithSettings and then below make the function return an array of UserWithSettings